code/__DEFINES/DNA.dm

```diff
@@ -92,6 +93,23 @@
 #define DNA_SKIN_TONE_BLOCK 3
 #define DNA_UNI_IDENTITY_BLOCKS 7
 
+#define DNA_FEATURE_BLOCKS 15
+#define DNA_MCOLOR_BLOCK 1
+#define DNA_ETHCOLOR_BLOCK 2
+#define DNA_LIZARD_MARKINGS_BLOCK 3
+#define DNA_LIZARD_TAIL_BLOCK 4
+#define DNA_SNOUT_BLOCK 5
+#define DNA_HORNS_BLOCK 6
+#define DNA_FRILLS_BLOCK 7
+#define DNA_SPINES_BLOCK 8
+#define DNA_HUMAN_TAIL_BLOCK 9
```

Comment:
Bit nitpicky, but maybe reptilian and humanoid markings and tail?
Feel free to ignore it if you think this way is more clear, but human tail sounds odd to me.